web/src/views/DandisetLandingView/ShareDialog.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <v-dialog
+    v-if="meta && meta.name"
```

Comment:
Just curious, are you able to use the optional chaining operator in the template?

```suggestion
    v-if="meta?.name"
```
If so, I'd recommend we use this in the template wherever possible.

Reply:
I wish, unfortunately not :(

https://github.com/vuejs/vue/issues/11088